libretro/libretro-core.c

```diff
@@ -35,6 +35,50 @@ static char buf[64][4096] = { 0 };
 long microSecCounter=0;
 int cpuloop=1;
 
+#ifdef __WIN32__
+int numlock_original_state=0;
+
+#ifdef __WIN32__
+#include <windows.h>
+#else 
+#include <X11/Xlib.h>

```

Comment:
See above.

Reply:
Hmm you cannot assume x11 is available anywhere. Also it is inside a Win32 guard.